pytests/test_common.py

```diff
@@ -103,80 +103,88 @@ def test_module_import_microgrid_lifetime() -> None:
     assert lifetime_pb2_grpc is not None
 
 
-def test_module_import_microgrid_components() -> None:
+def test_module_import_microgrid_electrical_components() -> None:
     """Test that the modules can be imported."""
     # pylint: disable=import-outside-toplevel
-    from frequenz.api.common.v1.microgrid import components
+    from frequenz.api.common.v1.microgrid import electrical_components
```

Comment:
It is really weird that these tests don't fail too, but I guess this is for a separate investigation/PR.